test/jdk/java/util/logging/CheckZombieLockTest.java

```diff
@@ -241,10 +241,11 @@ private static void testFileHandlerReuse(File writableDir) throws IOException {
             }
 
             if (supportsLocking) {
-                FileChannel fc = FileChannel.open(Paths.get(lock.getAbsolutePath()),
+                handler2 = null;
+                try (FileChannel fc = FileChannel.open(Paths.get(lock.getAbsolutePath()),
                     StandardOpenOption.CREATE_NEW, StandardOpenOption.APPEND,
-                    StandardOpenOption.WRITE);
-                try {
+                    StandardOpenOption.WRITE)) {
```

Comment:
Changing this to use try-with-resources looks good. In passing I wonder why it calls lock.getAbsolutePath(), it's okay to call open with a relative path. Also, if you change this test to use import static then you could change this line to:

`try (FileChannel fc = FileChannel.open(Path.of(lock), CREATE_NEW, APPEND, WRITE))`

Reply:
Good idea. Done.